IdentityServer4.Contrib.RedisStore/Stores/PersistedGrantStore.cs

```diff
@@ -31,7 +31,7 @@ public PersistedGrantStore(RedisMultiplexer<RedisOperationalStoreOptions> multip
             this.options = multiplexer.RedisOptions;
             this.database = multiplexer.Database;
             this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
-            this.clock = clock;
+            this.clock = clock ?? throw new ArgumentNullException(nameof(logger));
```

Comment:
One final thing @RyanWChild, it's "clock" not "logger", anyway I think it's not necessary to throw that exception.

Reply:
Thanks you !